### PR TITLE
Add missing repo, update iohk-monitoring dep

### PR DESCRIPTION
**Pull request by edsko**
PSA: Updating cardano-ledger to recent MASTER does _not_ work
(cardano-ledger-test needs updating).

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -128,3 +128,9 @@ source-repository-package
   location: https://github.com/input-output-hk/cardano-ledger
   tag: a773ba71eb9a6fb91dda2cdf6c41b2280f2acf7d
   subdir: crypto/test
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-crypto/
+  tag: 4590efa638397e952a51a8994b5543e4ea3c1ecd
+  subdir: .
```
Comment (mrBliss) [inline cabal.project]: Oops, my bad, I removed that.

But why didn't CI warn us about this?

Comment (edsko) [reply cabal.project]: Does CI use cabal to build?

**Comment by edsko** [comment]
bors retry

**Comment by mrBliss** [comment]
C'mon bors, stop slowing us down :weary:

File: nix/.stack.nix/default.nix [hunk 2]
```diff
@@ -50,6 +50,7 @@
         cardano-prelude = ./cardano-prelude.nix;
         cardano-prelude-test = ./cardano-prelude-test.nix;
         cardano-crypto = ./cardano-crypto.nix;
+        cardano-crypto = ./cardano-crypto.nix;
```
Comment (cleverca22) [inline nix/.stack.nix/default.nix]: this duplicate `cardano-crypto` is breaking hydra (and by extension bors)

Comment (edsko) [reply nix/.stack.nix/default.nix]: Thanks for looking into this!

**Comment by edsko** [comment]
Thanks @cleverca22 :)

**Pull request merged**